README.md

```diff
@@ -510,6 +511,46 @@ data:
 You're welcome to raise issues related to feature requests, bugs, or give us general feedback on this project's GitHub Issues page. 
 The SAP BTP service operator project maintainers will respond to the best of their abilities. 
 
+[Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
+
+## Uninstalling the Operator
+
+To uninstall the operator, run the following command:
+`helm uninstall <release name> -n <name space>`
+
+Example: 
+
+ >   ```
+  >   helm uninstall sap-btp-operator -n sap-btp-operator
+
+In a successful outcome, Operator is uninstalled and all associated service instances and bindings are deleted.
+
+**Note:** Make sure to first address any possible issues that may arise during the deletion of service instances and bindings before you proceed with the deletion of the operator.
+
+
+#### Possible Issues
+
+   - The deletion of instances and bindings takes more than 5 minutes
+   
+     There are two scenarios and their respective resolutions:
+     
+      - If you get the notification:`Error: job failed: BackoffLimitExceeded`, manually re-trigger the delete process by running `helm uninstall <release name> -n <name space>`.
+      
+      - If you get the following error: `Warning: Hook pre-delete failed: jobs.batch "pre-delete-job" already exists.` please wait for the job to finish.
+        To verify that the job has finished, verify that there is no pre-delete job in the cluster logs.
```

Comment:
Tal task: check for kubectl command to see the running jobs

Reply:
finished = not visible / status completed

Reply:
after the job finished - he can re-trigger the uninstall

Reply:
re-run the uninstall until successful run

Reply:
➜  ~ git:(master) ✗ kubectl get jobs --all-namespaces                   
No resources found

This means no job is running